cdc/owner.go

```diff
@@ -317,12 +318,34 @@ func (c *changeFeed) banlanceOrphanTables(ctx context.Context, captures map[stri
 	}
 }
 
-func (c *changeFeed) applyJob(job *timodel.Job) error {
+func (c *changeFeed) checkJob(job *timodel.Job) (skip bool) {
+	switch job.Type {
+	case timodel.ActionCreateTable:
+		tableInfo := entry.WrapTableInfo(job.BinlogInfo.TableInfo)
+		log.Warn("this table is not eligible to duplicate", zap.Reflect("job", job))
```

Comment:
This warning should not be logged when `tableInfo.ExistTableUniqueColumn()` is true